packages/service-library/src/servicelib/aiohttp/long_running_tasks/_server.py

```diff
@@ -31,6 +40,49 @@ async def _wrap(request: web.Request):
     return _wrap
 
 
+async def start_long_running_task(
```

Comment:
I was surprised not to see a test for this

Reply:
good point. it was partially moved from the webserver. will change it. good catch!

Reply:
well actually it is used by conftest.py so it is tested. but I will add an explicit one